change/@ni-nimble-components-5067d35f-244c-42ff-b498-4329212f03ce.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm still seeing some unexpected zoom behavior when using the touchpad on my macbook. It's your team's call whether to fix these now, in a future PR, or never, but I thought I'd point it out:

1. If I scroll (drag with two fingers) on a wafer map at 1x zoom, nothing happens. The page doesn't scroll but the wafer map doesn't zoom.
2. If I zoom in first (using one of the methods below) and then scroll by dragging two fingers, the wafer map does zoom.
3. If I momentum scroll (flick with two fingers), the wafer map does zoom.
4. If I hold down Control while dragging with two fingers (zoom), the wafer map does zoom.
5. If I pinch with two fingers, the wafer map doesn't zoom.

I would expect all of these to behave the same and cause the wafer map to zoom.

Reply:
We will address these concerns in a later PR since this one has been blocking others for some time and it would be more productive to close it and just open a bug on it.